ExtraEnemyCustomization/Customizations/Models/MarkerCustom.cs

```diff
@@ -59,6 +60,9 @@ public void OnPrefabBuilt(EnemyAgent agent)
 
         public void OnSpawned(EnemyAgent agent)
         {
+            if (AllowMarkingOnHibernate)
+                agent.ScannerData.m_soundIndex = 0; //I know... this is such a weird way to do it...
```

Comment:
10cc loves to name their variables and use them in confusing ways - `m_soundIndex` looks like the only way to make them markable.